libc/test/src/sched/CMakeLists.txt

```diff
@@ -8,6 +8,8 @@ add_libc_unittest(
     affinity_test.cpp
   DEPENDS
     libc.include.sched
```

Comment:
Remove `libc.include.sched` from the cmake for each file you removed `#include <sched.h>` from.